tests/cpu/test_track_params_estimation.cpp

```diff
@@ -46,18 +47,23 @@ TEST(track_params_estimation, helix_negative_charge) {
         pos, time, vector::normalize(mom), q / vector::norm(mom), &B);
 
     // Make three spacepoints with the helix
-    spacepoint_collection_types::host spacepoints;
-    spacepoints.push_back({hlx(50 * unit<scalar>::mm), {}});
-    spacepoints.push_back({hlx(100 * unit<scalar>::mm), {}});
-    spacepoints.push_back({hlx(150 * unit<scalar>::mm), {}});
+    measurement_collection_types::host measurements(&host_mr);
+    edm::spacepoint_collection::host spacepoints{host_mr};
+    measurements.resize(3);
+    spacepoints.reserve(3);
+    spacepoints.push_back({0, hlx(50 * unit<scalar>::mm), 0.f, 0.f});
```

Comment:
What is the index of 0,1,2?

Reply:
Probably I could simplify the test here a bit. 🤔

I wanted the test spacepoints to point at "valid" measurements. That's why I resized the measurement container to `3`.

But no algorithmic code tries to access measurements through these spacepoints in this test. So even creating the measurement collection is really unnecessary in the end.

So `0`, `1` and `2` are meant to be the indices of the elements in `measurements`. Though in this test it really doesn't matter...

Reply:
Never mind, let's not "simplify" this test.

The track parameter estimation does access one of the measurements. It's fine to have dummy values in the measurements, but it's still best to have existing objects for all three spacepoints in memory.